src/mod_zone_difficulty_scripts.cpp

```diff
@@ -265,7 +265,7 @@ void ZoneDifficulty::SendWhisperToRaid(std::string message, Creature* creature,
  */
 void ZoneDifficulty::GrantHardmodeScore(Map* map, uint32 type)
 {
-    if (!map || map < 0 || type < 0 || type > 255)
+    if (!map || type < 0 || type > 255)
```

Comment:
its an uint32 (unsignaled integer), so type can never be < 0

Reply:
Makes sense, i'll fix it with the next push. Thanks for checking!

Reply:
Done a quick glance through the file and it seems this check is present elsewhere (checking for negative uints)

Reply:
I can find only one more and that one was picked from the AutoBalance module. I don't dare changing it tbh.